src/anyon/qpu_interface.jl

```diff
@@ -279,7 +294,11 @@ function get_status(client::Client,circuitID::String)::Status
     end
```

Comment:
We should also have a test if an invalid type is returned. (Maybe we add a type in the future and the Snowflake version is old)